Dockerfile

```diff
@@ -59,8 +59,7 @@ RUN install2.r --error --deps TRUE \
     pheatmap \
     Rtsne \
     umap  \
-    VennDiagram \
-    org.Mm.eg.db
```

Comment:
We will want to install this still, I think this is an indication that the branch is out of date? I think this PR predated Jackie's `up-to-date-with-master` changes: https://github.com/AlexsLemonade/refinebio-examples/pull/221

So we should be cognizant of that before we merge with master we need to resolve this.

Reply:
Ah this PR is actually up to date with master @cansavvy but as noted in this PR's original comment, I had the question of: `Should we be adding the individual org.Dr.eg.db organism package to the Dockerfile?`

and also began getting the following error when building the Docker image with `org.Mm.eg.db` being installed:
```
The downloaded source packages are in
        ‘/tmp/downloaded_packages’
Error: package ‘org.Mm.eg.db’ is not available (for R version 4.0.2)
In addition: Warning message:
package ‘org.Mm.eg.db’ is not available (for R version 4.0.2) 
```

Reply:
I think it needs to be installed with Bioconductor.

Reply:
I think given the purposes of just making sure those of us developing the material are using all the same packages, it makes sense to have `org.Dr.eg.db` added to the Dockerfile. 

Did you have some thoughts about why we shouldn't add it?

Reply:
Ah okay I got it to build successfully. I have no arguments on why it shouldn't be added here so I added both the mouse and zebrafish annotation packages 👍 (will push Docker image after PR is merged as instructed in `CONTRIBUTING.md`).